modules/openapi-generator/src/test/java/org/openapitools/codegen/typescript/fetch/TypeScriptFetchModelTest.java

```diff
@@ -95,7 +95,7 @@ public void simpleModelTest() {
         final CodegenProperty property4 = cm.vars.get(3);
         Assert.assertEquals(property4.baseName, "birthDate");
         Assert.assertEquals(property4.complexType, null);
-        Assert.assertEquals(property4.dataType, "Date");
+        Assert.assertEquals(property4.dataType, "string");
```

Comment:
this should be `Date`, since typescript-fetch implements date deserialization:
https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/resources/typescript-fetch/modelGeneric.mustache#L66